<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrdinationAnalysisDlg</class>
 <widget class="QDockWidget" name="OrdinationAnalysisDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>199</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Ordination analysis</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblEmbeddingMethod">
        <property name="text">
         <string>Embedding method:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cboClusteringMethod">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Contiguous isomap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nonmetric MDS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Isomap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Principal coordinate analysis</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblNeighborhoodValue">
        <property name="text">
         <string># neighbours (k):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="spinNeighborhoodValue">
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
          <property name="value">
           <double>7.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblIterations">
        <property name="text">
         <string>Max. iterations:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spinMaxIter">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblNeighbourhoodMethod">
        <property name="text">
         <string>Neighbourhood method:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboNeighbourhoodMethod">
        <item>
         <property name="text">
          <string>Fixed number of neighbours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sample dissimilarity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quantitative factor</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblNeighbourhoodField">
        <property name="text">
         <string>Field:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cboNeighbourhoodField"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnRun">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
